vdu-profile.sh

```diff
@@ -27,6 +24,10 @@ done
 # Wait for generated machineconfig to be applied
 oc wait --timeout=20m --for=condition=updated=true mcp master
 
+oc apply -f ./vdu/01-namespaces.yaml
```

Comment:
Let's rename the yamls to match the order in we apply them, and maybe also add a note in the script explaining why we're creating node-tunning the first one